_posts/2023/2023-02-27-consider-find_each-for-active-record-assocation-loops.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Consider find_each for looping through Active Record associations"
+description: "Rarely used, but protects you against foot shooting"
+layout: article
+category: ruby
+image:
+  base: '2023/consider-find_each-for-looping-through-active-record-associations'
+  alt: "Light loops"
+  credit: "Nareeta Martin"
+  source: "https://unsplash.com/photos/za2gMSfLmzU"
+
+---
+
+The standard method for enumerating through groups of objects is `each`, both through arrays in Ruby and through Active Record models in Rails.
```

Comment:
I would bout "both through arrays....Rails" after "group of objects", therefore ending the sentence with: "is `each`".

Reply:
```suggestion
The standard method for enumerating through groups of objects, both through arrays in Ruby and through Active Record models in Rails, is `each`.
```